src/metabase/public_settings.clj

```diff
@@ -1057,6 +1057,17 @@ See [fonts](../configuring-metabase/fonts.md).")
   :default    nil
   :type       :json)
 
+(defsetting bug-report-enabled
+  (deferred-tru "Enable bug report submissions.")
+  :visibility :public
+  :export?    false
+  :type       :boolean
+  :default    false
+  :setter     :none
+  :getter     (fn []
+                (boolean (env/env :mb-bug-reporting-enabled)))
```

Comment:
These 2 lines are unnecessary, and you can remove them because the environment lookup behavior will happen automatically by default. see: https://github.com/metabase/metabase/blob/master/src/metabase/models/setting.clj#L1162-L1172

Reply:
Looking at this again, I see that `:mb-bug-reporting-enabled` is different than `bug-report-enabled`. This will work by default if the name of the settings is `bug-reporting-enabled`. We usually don't allow a mismatch between the name of the defsetting and the env var used to control them.

Reply:
so if the env var to control this should be `MB_BUG_REPORTING_ENABLED`, I think this defestting should be called `bug-reporting-enabled`.